README.md

```diff
@@ -9,7 +9,7 @@ Please note that this is an Open
 release for **developers** and **power users** [only](#users).
 
 **Users** *should wait* for our 
-[Open Beta](https://github.com/ComputationalRadiationPhysics/picongpu/issues/milestones)
+[Open Beta](https://github.com/ComputationalRadiationPhysics/picongpu/issues?milestone=2)
```

Comment:
no, should like to the overview page of all milestones (to show the progress):
https://github.com/ComputationalRadiationPhysics/picongpu/milestones